### PR TITLE
Additional newlines in messages

### LINKED ISSUE #2481: Sending mail inserts newlines
Write a mail:
```
⏮ Last week:
- More 18 reviews and polishing
- Went through notifications
```

Send it, and see it formatted like this:
```
⏮ Last week:

- More 18 reviews and polishing

- Went through notifications
```

Priority cause it changes mail text unexpectedly. :)

### PR DESCRIPTION
**Pull request by kesselb**
Fix #2481 

Test steps are at https://github.com/nextcloud/mail/issues/2481#issuecomment-896327822

### CODE REVIEW

File: lib/Service/MailTransmission.php [hunk 1]
```diff
@@ -178,7 +179,8 @@ public function sendMessage(NewMessageData $messageData,
 		$mail = new Horde_Mime_Mail();
 		$mail->addHeaders($headers);
 		if ($messageData->isHtml()) {
-			$mail->setHtmlBody($message->getContent());
+			$mail->setHtmlBody($message->getContent(), null, false);
+			$mail->setBody(Horde_Text_Filter::filter($message->getContent(), 'Html2text', array('callback' => [$this, 'htmlToTextCallback'])));
```
Comment (ChristophWurst) [inline lib/Service/MailTransmission.php:183]:
```suggestion
			$mail->setBody(Horde_Text_Filter::filter($message->getContent(), 'Html2text', ['callback' => [$this, 'htmlToTextCallback']]));
```

**Review by ChristophWurst — APPROVED** [review]
Code looks good, didn't test

**Comment by GretaD** [comment]
I can still reproduce the new lines on plain text, but not on rich text

**Review by GretaD — APPROVED** [review]
all good now :+1:

**Pull request merged**